src/ucp/api/ucp_def.h

```diff
@@ -43,6 +43,15 @@ typedef struct ucp_tag_recv_info         ucp_tag_recv_info_t;
  */
 typedef struct ucp_context               *ucp_context_h;
 
+/**
+ * @ingroup UCP_WORKER_PARAMS
```

Comment:
it was not defined. we need to talk so you can explain me when and how to define those